apps/remote_control/lib/lexical/remote_control/code_intelligence/entity.ex

```diff
@@ -112,6 +113,15 @@ defmodule Lexical.RemoteControl.CodeIntelligence.Entity do
     end
   end
 
+  defp resolve({:module_attribute, attr_name}, node_range, analysis, position) do
+    current_module =
+      analysis
+      |> RemoteControl.Analyzer.aliases_at(position)
+      |> Map.get(:__MODULE__)
```

Comment:
```suggestion
   {:ok, current_module} = RemoteControl.Analyzer.expand_alias([:__MODULE__], analysis, position)
```

Reply:
does this work? I could have _sworn_ I tried this